packages/terra-icon/README.md

```diff
@@ -10,10 +10,22 @@ The terra-icon component is used to visually represent a literal or symbolic obj
 - [Documentation](https://engineering.cerner.com/terra-ui/components/terra-icon/icon/icon)
 - [LICENSE](#license)
 
-## Getting Started
+## Using Terra Icon
 
 - Install from [npmjs](https://www.npmjs.com): `npm install terra-icon`
 
+## Adding new icons from the One Cerner Style Icons repo
+
+    npm run migrate-ocs-icons
```

Comment:
nit: should this be in a code block (```)

Reply:
Nope, indenting by one tab is the same effect. [Check it out in action](https://github.com/cerner/terra-core/blob/update-script-instructions/packages/terra-icon/README.md).